java/src/jmri/jmrit/beantable/LRouteTableAction.java

```diff
@@ -970,7 +970,7 @@ public void actionPerformed(ActionEvent e) {
             tab2.add(new JLabel(rbx.getString("OutputTitle")));
             _outputAllButton = new JRadioButton(Bundle.getMessage("All"), true);
             JRadioButton includedOutputButton = new JRadioButton(Bundle.getMessage("Included"), false);
-            tab2.add(makeShowButtons(_outputAllButton, includedOutputButton, null, Bundle.getMessage("Show")));
+            tab2.add(makeShowButtons(_outputAllButton, includedOutputButton, null, "Show")); // NOI18N
```

Comment:
The NOI18N market indicates this *should never be translated*, not that a translation is missing. Are you sure you want to mark this as that?